### PR TITLE
Simplify BitUtils methods

### PR DESCRIPTION
**Pull request by grkvlt**
Couldn't help it. All tests still pass.

### CODE REVIEW

File: utils/common/src/main/java/org/apache/brooklyn/util/math/BitUtils.java [hunk 1]
```diff
@@ -18,25 +18,22 @@
...
      * reversed yields 1 = 0b00000001 = bit list {1,0,0,0,0,0,0,0} */
     public static byte reverseBitSignificance(byte b) {
-        int result = 0;
-        for (int i=0; i<8; i++) {
-            result <<= 1;
-            if ((b&1)==1) result++;
-            b >>= 1;
-        }
-        return (byte)result;
+        return (byte) (Integer.reverse(b) >> 24);
```
Comment (neykov) [inline utils/common/src/main/java/org/apache/brooklyn/util/math/BitUtils.java]: Wow [`Integer.reverse`](https://github.com/openjdk-mirror/jdk7u-jdk/blob/master/src/share/classes/java/lang/Integer.java#L1204) does some interesting voodoo magic :).
Should this be unsigned shift?

**Comment by aledsage** [comment]
@grkvlt can you respond to @neykov's question "can you confirm signed vs unsigned shift", and then good to merge.

File: utils/common/src/main/java/org/apache/brooklyn/util/math/BitUtils.java (hunk 1 above)
Comment (grkvlt) [inline utils/common/src/main/java/org/apache/brooklyn/util/math/BitUtils.java]: No. The following tests should convince you:
```
        Assert.assertEquals(BitUtils.reverseBitSignificanceInByte(0b00000001), (byte)0b10000000);
        Assert.assertEquals(BitUtils.reverseBitSignificanceInByte(0x0f), (byte)0xf0);
```

**Comment by neykov** [comment]
thanks @grkvlt, merging.

**Pull request merged**